packages/block-editor/src/utils/transform-styles/index.js

```diff
@@ -19,6 +19,7 @@ function transformStyle(
 		return css;
 	}
 
+	const postcssFriendlyCSS = css.replace( ':where(body)', 'body' );
```

Comment:
Would `:where(body)` ever be chained after another rule, i.e. `.my-class-name:where(body)`? If so, the result might not be valid CSS. I imagine that would be very edge-case territory as it's probably quite unlikely anyone would do that?

Reply:
Hmm yeah I'm not sure that will be much of a problem, because any unchanged selector with `:where(body)` would get the scoping selector prepended to it, so it would only work if the scoping selector happened to be attached to the `html` element. That's pretty unlikely to happen 😅